Source/WebKit/UIProcess/Extensions/Cocoa/WebExtensionSidebarCocoa.mm

```diff
@@ -403,6 +451,20 @@ - (void)viewWillDisappear
     m_children.remove(child);
 }
 
+void WebExtensionSidebar::didReceiveUserInteraction()
+{
+    auto currentTab = tab();
+    auto currentContext = extensionContext();
+
+    RELEASE_LOG_ERROR_IF(!isOpen(), Extensions, "didReceiveUserInteraction was called on a sidebar object which is not open");
+    RELEASE_LOG_ERROR_IF(!currentTab, Extensions, "didReceiveUserInteraction was called on a sidebar which has no associated tab");
```

Comment:
Should these just be ASSERTs?

Reply:
They definitely could be, I don't really have a strong preference either way. It seems like this logging could possibly be useful for debugging sidebar issues that could theoretically occur in the wild, but it was more of a "why not log something" situation when I wrote it.